opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -352,6 +353,8 @@ namespace Opm
                          SimulationDataContainer& simulatorstate,
                          WellStateFullyImplicitBlackoil& wellstate)
     {
+        // gives a dummy dynamic_list_econ_limited
+        DynamicListEconLimited dummy_list_econ_limited;
```

Comment:
> @joakim-hove do you know?

The current (in flux) I/O facilities are very much in flux so I am not entirely sure; but at least previously they  write wells/connections to the restart file as closed/open depending on the `WellState` object. 

As I see it the Right&trade; way to do this is that the simulator initialized/assigns the `data:Wells` structures correctly with open/shut status of wells and connections, and then that stumbles out to disk.

What I do know with ~ 100% certainty is that *the published part* of the restart format does *not* have any option set a flag indicating *why* a particular well/connection is closed.